l10n_ch_account_reconcile_esr/i18n/it.po

```diff
@@ -0,0 +1,43 @@
+# Translation of Odoo Server.
+# This file contains the translation of the following modules:
+# 	* l10n_ch_account_reconcile_esr
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: Odoo Server 10.0+e\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2019-05-13 14:39+0000\n"
+"PO-Revision-Date: 2019-05-13 16:41+0200\n"
+"Last-Translator: Timon Tschanz <timon.tschanz@camptocamp.com>\n"
+"Language-Team: Italian\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: nplurals=2; plural=(n != 1);\n"
+"Language: it\n"
+
+#. module: l10n_ch_account_reconcile_esr
+#: model:ir.model,name:l10n_ch_account_reconcile_esr.model_account_bank_statement_line
+msgid "Bank Statement Line"
+msgstr "Linea estratto conto"
+
+#. module: l10n_ch_account_reconcile_esr
+#. openerp-web
+#: code:addons/l10n_ch_account_reconcile_esr/static/src/xml/account_reconciliation.xml:9
+#, python-format
+msgid "ESR reconciliation"
+msgstr "Riconciliazione PVR"
+
+#. module: l10n_ch_account_reconcile_esr
+#. openerp-web
+#: code:addons/l10n_ch_account_reconcile_esr/static/src/xml/account_reconciliation.xml:9
+#, python-format
+msgid "Let odoo try to reconcile entries for the user using esr refs"
+msgstr ""
```

Comment:
@eLBati may I ask your help here? :pray: 
Does this sound good to you? "Consenti a Odoo di riconciliare le linee usando i riferimenti ESR"

Reply:
Io metterei "Consenti a odoo di provare a riconciliare le registrazioni usando i riferimenti ESR"

Reply:
@tschanzt pick this :wink: 
@eLBati tnx!